packages/bubble-dev/start-preset/src/pkg.ts

```diff
@@ -62,44 +62,51 @@ export const Pkg = (replacers?: TReplacers) => (packagePath: string, shouldAddMi
     }
 
     let type = null
+    let name = null
 
-    if (templateNames.length === 1) {
-      type = templateNames[0]
+    if (Boolean(templateName) && templateNames.includes(templateName)) {
+      type = templateName
     } else {
-      const result = await prompts(
+      if (templateNames.length === 1) {
+        type = templateNames[0]
+      } else {
+        const result = await prompts(
+          {
+            type: 'select',
+            name: 'type',
+            message: 'Choose package type',
+            choices: templateNames.map((templateName) => ({
+              title: templateName,
+              value: templateName,
+            })),
+          },
+          {
+            onCancel: () => {
+              throw 'canceled'
+            },
+          }
+        ) as { type: string }
+
+        type = result.type
+      }
+
+      const packageNameResult = await prompts(
```

Comment:
Why this change is needed? It doesn't seem you're using `packageNameResult` anywhere else, besides getting the `name` out of it lower down in the code?

Reply:
I just made it the same as in type both for keeping consistency and to allow me to define the name variable earlier (as type was defined)